tools/acle_template.tex

```diff
@@ -57,104 +59,112 @@
+        \hspace{1.0cm} \includegraphics[height=0.6cm]{"./tools/Arm-logo-blue-RGB.pdf"}
+        \begin{minipage}[t][1.2cm][t]{\linewidth}
+            \phantom{empty}
+        \end{minipage}
+        \newline
+        \begin{minipage}[t]{4.0cm}
+            \phantom{empty}
+        \end{minipage}
+        \begin{minipage}[t]{14.0cm}
+            {\huge \bfseries  $title$}\\[2ex]
+            {\large Release Date: $date-of-issue$}\\[2ex]
+            {\large \bfseries Version: $version$}\\[3ex]
+        \end{minipage}
+        \newline
+        \begin{minipage}[t][0.47cm][t]{\linewidth}
+            \phantom{empty}
+        \end{minipage}
+        \newline
+        \begin{minipage}[t]{4.0cm}
+            \phantom{empty}
+        \end{minipage}
+        \begin{minipage}[t]{16.0cm}
+            \begin{minipage}[t]{9.0cm}
+                \textbf{$copyright-text$}\\[$if(landscape)$7ex$else$1ex$endif$]
+            \end{minipage}
+        \end{minipage}
+    \end{flushleft}
+    \begin{figure}[hbt!]
+
+        \makebox[\paperwidth]{
+          $if(landscape)$\begin{subfigure}[b!]{13.8cm}$endif$
+            \begin{tikzpicture}[scale=\paperwidth/$if(landscape)$16.0cm$else$8.0cm$endif$,samples=200]
+                \draw[line width=0.0pt,color=white,fill=armblue] (5.005,0) rectangle (5.995,7);
+                \draw[line width=0.0pt,color=white,fill=armblue] (0,4.995) rectangle (8,4.005);
+                \foreach \y in {0,...,7} {
+                    \draw[lightgrey, thin] (0,\y) -- (8,\y);
+                }
+                \foreach \x in {1,...,7} {
+                    \draw[lightgrey, thin] (\x,0) -- (\x,7);
+                    \foreach \y in {1,...,7} {
+                        \draw[grey, thick] (\x,\y - 0.05) -- (\x,\y + 0.05);
+                        \draw[grey, thick] (\x - 0.05,\y) -- (\x + 0.05,\y);
+                        \draw[white, thick] (\x - 0.01,\y + 0.01) rectangle (\x + 0.01,\y - 0.01);
+                    }
+                }
+            \end{tikzpicture}
+          $if(landscape)$\end{subfigure}$endif$
+          $if(landscape)$\begin{subfigure}[b!]{13.8cm}
+            \begin{tikzpicture}[scale=\paperwidth/16.0cm,samples=200]
+                \draw[line width=0.0pt,color=white,fill=armblue] (5.005,0) rectangle (5.995,7);
+                \draw[line width=0.0pt,color=white,fill=armblue] (0,4.995) rectangle (8,4.005);
+                \foreach \y in {0,...,7} {
+                    \draw[lightgrey, thin] (0,\y) -- (8,\y);
+                }
+                \foreach \x in {0,...,7} {
+                    \draw[lightgrey, thin] (\x,0) -- (\x,7);
+                    \foreach \y in {1,...,7} {
+                        \draw[grey, thick] (\x,\y - 0.05) -- (\x,\y + 0.05);
+                        \draw[grey, thick] (\x - 0.05,\y) -- (\x + 0.05,\y);
+                        \draw[white, thick] (\x - 0.01,\y + 0.01) rectangle (\x + 0.01,\y - 0.01);
+                    }
+                }
             \end{tikzpicture}
```

Comment:
Just... wow! You are drawing the grid in the front page!!! Am I right? Please add comments specifying which bits represent the grid of the page, this is not trivial LaTeX code.

Reply:
Yes this is based on the code used in the example document: https://eu-gerrit-1.euhpc.arm.com/plugins/gitiles/dsg/apd/morello/morelloie/+/refs/heads/dev/docs/user_guide/preamble.tex#89